sources/packages/web/src/components/common/LocationSummary.vue

```diff
@@ -1,40 +1,40 @@
 <template>
   <!-- This component is shared between ministry and student users -->
-  <v-row>
-    <v-col cols="8">
-      <p class="category-header-large color-blue">
-        All Locations({{ institutionLocationList.length ?? 0 }})
-      </p>
-    </v-col>
-    <v-col cols="4" v-if="clientType === ClientIdType.Institution">
+  <body-header
+    title="All locations"
+    class="m-1"
+    :recordsCount="institutionLocationList.length"
+    ><template #actions>
       <v-btn
+        v-if="clientType === ClientIdType.Institution"
```

Comment:
Not a part of the PR. But we will have to eventually eliminate the logic which is based on the client type on the vue components. for e.g. this button can be loaded based on a generic property like viewOnly which will be set by the view/parent using this component.

Adding a TODO here will help to identify those.